tests/System.IO.Compression.Tests/BrotliPerfomanceTests.cs

```diff
@@ -106,22 +105,22 @@ public enum CompressionType
             NormalData
         }
 
-        [Benchmark]
+        [Benchmark(InnerIterationCount = 10000)]
         [InlineData(CompressionType.CryptoRandom)]
         [InlineData(CompressionType.RepeatedSegments)]
         [InlineData(CompressionType.NormalData)]
-        public void Decompress(CompressionType type)
+        public void DecompressUsingStream(CompressionType type)
         {
             string testFilePath = CreateCompressedFile(type);
-            int bufferSize = 1024 * 32;
+            int bufferSize = 1024;
             int retCount = -1;
             var bytes = new byte[bufferSize];
             using (MemoryStream brStream = new MemoryStream(File.ReadAllBytes(testFilePath)))
                 foreach (var iteration in Benchmark.Iterations)
                     using (iteration.StartMeasurement())
-                        for (int i = 0; i < 10000; i++)
+                        for (int i = 0; i < Benchmark.InnerIterationCount; i++)
```

Comment:
Move `int retCount = -1;` inside the loop.
Right now, you will enter the `while (retCount != 0)` loop once. After you break (after first iteration of benchmark), retCount will be 0 and you will not enter the loop again for subsequent Benchmark.Iterations.